provisioning_client/tests/prov_transport_amqp_common_ut/prov_transport_amqp_common_ut.c

```diff
@@ -456,6 +456,63 @@ static void on_umock_c_error(UMOCK_C_ERROR_CODE error_code)
 
 static TEST_MUTEX_HANDLE g_testByTest;
 
+char* umock_stringify_BINARY_DATA(const BINARY_DATA* value)
```

Comment:
Same comments as before for BINARY_DATA.

Reply:
see above